docs/tutorial/get-started.md

```diff
@@ -39,773 +54,329 @@ brew install hdf5
+    vald-discoverer-7f9f697dbb-q44qh           1/1     Running     0          7m11s
+    vald-lb-gateway-6b7b9f6948-4z5md           1/1     Running     0          7m12s
+    vald-lb-gateway-6b7b9f6948-68g94           1/1     Running     0          6m56s
+    vald-lb-gateway-6b7b9f6948-cvspq           1/1     Running     0          6m56s
+    vald-manager-index-74c7b5ddd6-jrnlw        1/1     Running     0          7m12s
+    ```
+
+## Running Exapmle Code
+
+In this chaptor, you will execute insert vectors, search vectors, and delete vectors to your Vald cluster using the exmaple code.<br>
+The [fashion-mnist](https://github.com/zalandoresearch/fashion-mnist) is used as a dataset for indexing and search query.
+
+The example code is implemented Go and using [vald-client-go](https://github.com/vdaas/vald-client-go), one of the official Vald client library,  for requesting to Vald cluster.
+Vald provides multiple language client libraries such as Go, Java, Node.js, Python, and so on.
+If you are interested in, please refer to [SDKs](../../user-guides/sdks).<br>
 
 1. Port Forward
 
-   ```bash
-   kubectl port-forward deployment/vald-lb-gateway 8081:8081
-   ```
-
-1. Download dataset
+    At first, port-foward is required to make request from your local environment possible.
 
-   In this tutorial. we use [fashion-mnist](https://github.com/zalandoresearch/fashion-mnist) as a dataset for indexing and search query.
+    ```bash
+    kubectl port-forward deployment/vald-lb-gateway 8081:8081
+    ```
 
-   ```bash
-   # move to working directory
-   cd example/client
+1. Download dataset
 
-   # download fashion-mnist testing dataset
-   wget http://ann-benchmarks.com/fashion-mnist-784-euclidean.hdf5
-   ```
+    Download [fashion-mnist](https://github.com/zalandoresearch/fashion-mnist) that is used as dataset for indexing and search query.
 
-1. Running example
+    ```bash
+    # move to working directory
+    cd example/client
 
-   Vald provides multiple language client libraries such as Go, Java, Node.js, Python, and so on.<br>
-   In this example, the fashion-mnist dataset will insert into the Vald cluster and perform a search using [vald-client-go](https://github.com/vdaas/vald-client-go).
+    # download fashion-mnist testing dataset
+    wget http://ann-benchmarks.com/fashion-mnist-784-euclidean.hdf5
+    ```
 
-   We use [`example/client/main.go`](https://github.com/vdaas/vald/blob/master/example/client/main.go) to run the example.
-   This will execute 6 steps.
+1. Run
 
-   1. init
+    We use [`example/client/main.go`](https://github.com/vdaas/vald/blob/master/example/client/main.go) to run the example.<br>
+    This example will insert and index 400 vectors into the Vald from the fashion-mnist dataset via grpc.
+    And then after waiting for indexing, it will request for searching nearest vector at 10 times.
+    You will get the 10 nearest neighbor vectors for each search query.<br>
+    Run example codes by executing below command.
```

Comment:
```suggestion
    We use [`example/client/main.go`](https://github.com/vdaas/vald/blob/master/example/client/main.go) to run the example.<br>
    This example will insert and index 400 vectors into the Vald from the fashion-mnist dataset via gRPC.
    And then after waiting for indexing, it will request for searching the nearest vector 10 times.
    You will get the 10 nearest neighbor vectors for each search query.<br>
    Run example codes by executing below command.
```